web/styles/app_components.css

```diff
@@ -736,11 +742,66 @@ div.overlay {
         }
     }
 
+    tbody > tr:nth-child(odd) > td {
+        background-color: hsl(0deg 0% 98%);
+    }
+
     /* Force the actions column to use the minimum space necessary */
     .actions {
         width: 1%;
         white-space: nowrap;
     }
+
+    & th,
+    td {
+        padding: 4px 5px;
+    }
+
+    & th {
+        text-align: left;
+    }
+
+    & td {
+        border-top: 1px solid hsl(0deg 0% 87%);
+    }
+}
+
+.table-bordered {
```

Comment:
This can be moved to `components.css` to avoid copying it again for portico pages.

Reply:
Probably better not to; I think we're more likely to fork these two table styles than further unify them in the ufutre.